shared/lib_battery.cpp

```diff
@@ -925,22 +928,25 @@ double voltage_vanadium_redox_t::voltage_model(double q0, double qmax, double I_
 	double SOC_use = q0 / qmax;
 	if (SOC_use > 1. - tolerance)
 		SOC_use = 1. - tolerance;
+	if (SOC_use == 0)
+	    SOC_use = 1e-3;
 
 	double A = std::log(std::pow(SOC_use, 2) / std::pow(1 - SOC_use, 2));
-
-	return _V_ref_50 + m_RCF * T * A + I_string * _R;
+	return _V_ref_50 + m_RCF * T * A + fabs(I_string) * _R;
```

Comment:
Can you add a comment here about how our voltage assumptions are different from the paper? (justifying the fabs call)

Reply:
Sure, good point!

Reply:
// In constrast to the V_stack + I_stack * R_specific in the paper which follows the convention of negative voltages,
// here the abs(I_stack) is used to allow both terms to move in same direction (https://github.com/NREL/ssc/issues/404)